fvm/utils/view.go

```diff
@@ -137,3 +170,19 @@ func (m *MapLedger) Delete(owner, key string) error {
 	delete(m.RegisterTouches, flow.RegisterID{Owner: owner, Key: key})
 	return nil
 }
+
+func (m *MapLedger) Payloads() []ledger.Payload {
+	ret := make([]ledger.Payload, 0, len(m.Registers))
+	for id, val := range m.Registers {
+		keyParts := []ledger.KeyPart{
+			ledger.NewKeyPart(0, []byte(id.Owner)),
+			ledger.NewKeyPart(2, []byte(id.Key)),
+		}
+
+		ret = append(
+			ret,
+			*ledger.NewPayload(ledger.NewKey(keyParts), ledger.Value(val)))
```

Comment:
It's better to make a `key := KeyFromRegister(id)`, so that we can hide the implementation details of those key parts.